docs/reference/style/pack.rst

```diff
@@ -140,20 +140,20 @@ direction of the parent's layout.
 The amount of space to allocate between the edge of the box, and the edge of the content
 in the box, in :ref:`CSS pixels <css-units>`.
```

Comment:
The current wording describes interior padding, which we have never supported, regardless of what the property was called.

```suggestion
The amount of space to allocate outside the edge of the widget, in :ref:`CSS pixels 
<css-units>`.
```

Reply:
Only thing I amended here is keeping "box" instead of "widget", to be consistent with how the page discusses geometry.

Reply:
The documentation was already a bit confusing here, because I think this is the only page where "box" is used synonymously with "widget" as opposed to the more specific meaning of "a flexbox container". But there's no need to fix that in this PR.